lib/array.js

```diff
@@ -35,17 +32,16 @@ const range = (
   return range;
 };
 
-const sequence = (
-  // Generate int array from sequence syntax
-  seq, // array
-  max // number, optional max
-  // Returns: array
-  // Example:
-  // list: sequence([81, 82, 83]) = [81, 82, 83]
-  // range from..to: sequence([81,,83]) = [81, 82, 83]
-  // range from..count: sequence([81, [3]]) = [81, 82, 83]
-  // range from..max-to: sequence([81, [-2]], 5) = [81, 82, 83]
-) => {
+// Generate int array from sequence syntax
+//   seq - array
+//   max - number, optional max
+// Returns: array
+// Example:
```

Comment:
Should we change this to 'Examples:' everywhere?
OR
We might use 'Example:' prefix for each example in order to cleanly separate them and parse/display properly later on.

I personally prefer the latter style, as it is more descriptive, understandable for the reader and supports multiline examples simply.
@SemenchenkoVitaliy @belochub @nechaido @tshemsedinov wdyt?